api/src/pcapi/routes/pro/users.py

```diff
@@ -160,9 +161,12 @@ def get_user_email_pending_validation() -> users_serializers.UserEmailValidation
 @spectree_serialize(on_error_statuses=[404], on_success_status=204, api=blueprint.pro_private_schema)
 def check_activation_token_exists(token: str) -> None:
     try:
-        users_repo.get_user_with_valid_token(token, [TokenType.RESET_PASSWORD], use_token=False)
+        token_utils.Token.load_and_check(token, token_utils.TokenType.RESET_PASSWORD)
     except users_exceptions.InvalidToken:
-        flask.abort(404)
+        try:
+            users_repo.get_user_with_valid_token(token, [TokenType.RESET_PASSWORD], use_token=False)
+        except users_exceptions.InvalidToken:
+            flask.abort(404)
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** ‏Même proposition pour le commentaire